### PR TITLE
Fix param error in ByteUtils#equals in java9 #1551

### PR DESCRIPTION
**Pull request by haishui126**
## Summary
Fix for issue https://github.com/ClickHouse/clickhouse-java/issues/1551

### CODE REVIEW

**Comment by chernser** [comment]
@haishui126 thank you for the contribution!
I see some steps have failed. We fixed the issue. Would you please pull latest changes to the branch to verify the fix in CI? 

Thank you in advance!

**Comment by haishui126** [comment]
> @haishui126 thank you for the contribution! I see some steps have failed. We fixed the issue. Would you please pull latest changes to the branch to verify the fix in CI?
> 
> Thank you in advance!

Hi @chernser, 
I have merged main into my branch. How can I verify my fix in CI? Workflow requires approval from a maintainer.

File: clickhouse-data/src/main/java9/com/clickhouse/data/ByteUtils.java
Comment (mshustov) [inline clickhouse-data/src/main/java9/com/clickhouse/data/ByteUtils.java:10]: @chernser I guess we don't have any tests for the methog

Comment (chernser) [reply clickhouse-data/src/main/java9/com/clickhouse/data/ByteUtils.java:10]: @mshustov I will take care of it this time.

**Comment by chernser** [comment]
@haishui126 Thank you! 
I see @mshustov triggered the CI (thank you) and it is passed.

**Review by chernser — APPROVED** [review]

**Pull request merged**
